### PR TITLE
README: Add instructions for running autoscaler tests under minikube

### PR DESCRIPTION
**Pull request by frobware**
Add broad instructions for running the cluster autoscaler e2e tests using a local minikube cluster.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,77 @@
...
+--- PASS: TestE2E (0.00s)
+PASS
+ok      github.com/openshift/cluster-api-actuator-pkg/pkg/e2e   0.037s
+```
+
+Adjust `-ginkgo.focus` as appropriate.
+
+### Cleanup
+
+The tests (during development) will / can fail and I use the following
```
Comment (vikaschoudhary16) [inline README.md]: s/I/@frobware? :)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,77 @@
 # cluster-api-actuator-pkg
 
 Shared packages for Cluster API actuators.
+
+## Runing the e2e tests locally using minikube
```
Comment (enxebre) [inline README.md]: how would this work without running the mao which is a requirement for cao and the clusteroperator CRD?

File: README.md (hunk 1 above)
Comment (frobware) [reply README.md]: Done in f60a9484...bef6ae54

File: README.md (hunk 2 above)
Comment (frobware) [reply README.md]: > how would this work without running the mao which is a requirement for cao and the clusteroperator CRD?

Made the comment more specific - now only caters for autoscaler tests.

**Comment by frobware** [comment]
/hold 

Updating instructions with output from running the individual commands.

**Pull request merged**